cypress/integration/sitemap-vrt/constants.ts

```diff
@@ -19,22 +19,44 @@ export const SITEMAP = [
   "/blog/2023-08-01-bringing-cohesion-to-the-twilio-product-suite-part-ii/",
   "/blog/2023-08-02-paste-newsletter/",
   "/components/account-switcher/",
+  "/components/account-switcher/api",
+  "/components/account-switcher/changelog",
   "/components/aspect-ratio/",
   "/components/aspect-ratio/api",
   "/components/aspect-ratio/changelog",
   "/components/anchor/",
   "/components/anchor/api",
   "/components/anchor/changelog",
   "/components/alert-dialog/",
+  "/components/alert-dialog/api",
+  "/components/alert-dialog/changelog",
   "/components/avatar/",
+  "/components/avatar/api",
+  "/components/avatar/changelog",
   "/components/breadcrumb/",
+  "/components/breadcrumb/api",
+  "/components/breadcrumb/changelog",
   "/components/badge/",
+  "/components/badge/api",
+  "/components/badge/changelog",
   "/components/callout/",
+  "/components/callout/api",
+  "/components/callout/changelog",
   "/components/code-block/",
+  "/components/code-block/api",
+  "/components/code-block/changelog",
   "/components/card/",
+  "/components/card/api",
+  "/components/card/changelog",
   "/components/chat-composer/",
+  "/components/chat-composer/api",
+  "/components/chat-composer/changelog",
   "/components/chat-log/",
+  "/components/chat-log/api",
+  "/components/chat-log/changelog",
   "/components/checkbox/",
+  "/components/checkbox/api",
+  "/components/checkbox/changelog",
```

Comment:
question (non-blocking): should we be VRTing changelog pages? Relatively unharmful even if one were to break, and the situation where they do is exceedingly rare

Reply:
I guess there's little harm in removing them, but not sure how we'd ever notice that they've broken in the off chance that they have? Would it be worth it in cypress runtime saved?

Reply:
Better to be safer than sorry. I'd rather not have a customer find a bug before us